examples/with-google-analytics/lib/gtag.js

```diff
@@ -1,17 +0,0 @@
-export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_ID
-
-// https://developers.google.com/analytics/devguides/collection/gtagjs/pages
-export const pageview = (url) => {
-  window.gtag('config', GA_TRACKING_ID, {
-    page_path: url,
-  })
-}
-
-// https://developers.google.com/analytics/devguides/collection/gtagjs/events
-export const event = ({ action, category, label, value }) => {
```

Comment:
Having this is a separate lib file was nice, in case you wanted to track events anywhere else in your application 👍